libalpm/Cargo.toml

```diff
@@ -12,7 +12,7 @@ An aspirationally safe wrapper around libalpm (the Arch Linux Package Manager).
 """
 
 [dependencies]
-alpm-sys = "0.1"
+alpm-sys = { git = "https://github.com/jameslzhu/alpm", branch = "master" }
```

Comment:
This should probably be reverted in the crates.io release.

Reply:
Yes, among other stuff. Now that I have approve from derekreery and everyone, I'll make actual changes. (Besides changes done in "-fork" to publish it, which will not be needed once we make upstream release anyway.)